ovs/lib/scheduledtask.py

```diff
@@ -263,11 +269,10 @@ def gather_scrub_work():
                 local_scrub_location = scrub_info[1]
```

Comment:
The local started scrub task should be handled the same way you process the remote tasks with these changes in place, because the local celery task can also remain in PENDING state if the local workers are stopped for whatever reason
Your ```result_set``` does not contain this local storagerouter